network/test/middleware_test.go

```diff
@@ -431,38 +462,6 @@ func (m *MiddlewareTestSuite) createOverlay(provider *testutils.UpdatableIDProvi
 	return overlay
 }
 
-// TestPingRawReception tests the middleware for solely the
```

Comment:
Why are these tests not needed anymore ?

Reply:
These four tests have been consolidated into the `TestPing`:
- `TestPingRawReception` tests payload of the received message, and `TestPing` now covers this evaluation [here](https://github.com/onflow/flow-go/pull/3675/files/9087a41417c995b2604600123ae027503859c407#diff-199e0c5e2e32815e4ac1058c9c089d4a9f8a7059f6399145c6156445b7bb3642R512).
- `TestPingTypeReception` tests type of the received payload, and `TestPing` now covers this evaluation [here](https://github.com/onflow/flow-go/pull/3675/files/9087a41417c995b2604600123ae027503859c407#diff-199e0c5e2e32815e4ac1058c9c089d4a9f8a7059f6399145c6156445b7bb3642R512). 
- `TestPingIDType` tests the origin id of the received message, and we have an enhanced testing of this (both type and value) [here](https://github.com/onflow/flow-go/pull/3675/files/9087a41417c995b2604600123ae027503859c407#diff-199e0c5e2e32815e4ac1058c9c089d4a9f8a7059f6399145c6156445b7bb3642R509) in `TestPing`. 
- `TestPingContentReceiption` tests the entire scope of the received message, which is now covered by `TestPing` [here](https://github.com/onflow/flow-go/pull/3675/files/9087a41417c995b2604600123ae027503859c407#diff-199e0c5e2e32815e4ac1058c9c089d4a9f8a7059f6399145c6156445b7bb3642R503-R512).